source/MapPanel.cpp

```diff
@@ -59,6 +59,15 @@ namespace {
 	const std::string SHOW_ESCORT_SYSTEMS = "Show escort systems on map";
 	const std::string SHOW_STORED_OUTFITS = "Show stored outfits on map";
 
+	// Struct to track per system how many pointers are drawn and still
+	// need to be drawn.
+	struct PointerDrawCount{
+		// Amount of systems already drawn.
```

Comment:
```suggestion
	class PointerDrawCount {
	public:
		// Amount of systems already drawn.
```

MZ doesn't like structs.

Reply:
I'm fine with their use when it is to provide named fields to what would otherwise be a pair or a tuple.

Reply:
Yeah same. We'll need to update the style guide at some point.